tests/fixtures/sample_lint_data.py

```
@@ -0,0 +1,439 @@
+            "linter": "flake8"
+        },
+        {
+            "file_path": "utils.py",
+            "line": 12,
+            "column": 15,
+            "rule_id": "F841",
+            "message": "local variable 'unused_var' is assigned to but never used",
+            "severity": "WARNING",
+            "linter": "flake8"
+        }
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)